### PR TITLE
Updating to RTM builds of xunit and Moq

### PR DESCRIPTION
**Pull request by pranavkm**

### CODE REVIEW

**Comment by pranavkm** [comment]
cc @ajaybhargavb

File: test/Microsoft.AspNetCore.Localization.FunctionalTests/project.json [hunk 1]
```diff
@@ -3,33 +3,24 @@
...
           "version": "1.0.0-*",
           "type": "platform"
-        },
-        "System.Diagnostics.Process": "4.1.0-*"
+        }
       }
     },
-    "net451": {
-      "frameworkAssemblies": {
-        "System.Runtime": ""
```
Comment (ajaybhargavb) [inline test/Microsoft.AspNetCore.Localization.FunctionalTests/project.json]: Was this intentional?

**Comment by ajaybhargavb** [comment]
:shipit:

File: test/Microsoft.AspNetCore.Localization.FunctionalTests/project.json (hunk 1 above)
Comment (pranavkm) [inline test/Microsoft.AspNetCore.Localization.FunctionalTests/project.json]: Yeah, it doesn't seem necessary.

**Pull request merged**